src/momento/responses/vector_index/data/search.py

```diff
@@ -24,15 +26,27 @@ class SearchResponse(VectorIndexResponse):
 class SearchHit:
     id: str
     distance: float
-    metadata: dict[str, str] = field(default_factory=dict)
+    metadata: dict[str, str | int | float | bool | list[str]] = field(default_factory=dict)
```

Comment:
Same comment here re: type alias.

Reply:
if it's just 2 places then i agree with the choice to bias towards IDE compat